app/controllers/application_controller.rb

```diff
@@ -2,11 +2,13 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
-  before_filter :get_next_meetup
+  before_filter :get_next_meetup, :get_events
 
   def get_next_meetup
     @next_meetup = Meetup.next
   end
 
-
+  def get_events
+    @events = Event.published
```

Comment:
I think that variable should be assigned in `HomeController` because it is used in `app/views/home/index.html.erb`. The other variable `@next_meetup` in the method up there is something that was previously used by the application layout so that why it's there.